app_config.py

```diff
@@ -1,14 +1,33 @@
 import os
 
+
+if (os.getenv('B2C_TENANT_NAME')
+    and os.getenv('SIGNUPSIGNIN_USER_FLOW') and os.getenv('EDITPROFILE_USER_FLOW')):
+    # This branch is for B2C. You can delete this branch if you are not using B2C.
```

Comment:
One more thing. Previously, this sample had a separate set of `app_config_b2c.py`. Therefore its usage would incur [renaming configuration files](https://learn.microsoft.com/en-us/azure/active-directory-b2c/configure-authentication-sample-python-web-app?tabs=linux#step-4-configure-the-sample-web-app) which was troublesome (and prone to error of accidentally committing the wrong file to override `app_config.py`).

This PR combines the previous `app_config_b2c.py` into `app_config.py`. Renaming is no longer needed. `README_b2c.md` is also combined into `README.md`.

One thing though. Since one `app_config.py` will contain both AAD and B2C settings, I choose to rename the previous B2C's `TENANT_NAME` into `B2C_TENANT_NAME`. You may want to update relevant docs, @pamelafox

Reply:
Good! We like the approach of using environment variables as configuration and having a single config. (as described in this manifesto: https://12factor.net/config)

I'm not sure I'll have time before I go OOO to update docs, we'll see.